index.js

```diff
@@ -609,7 +620,12 @@ function errorEventQueue(self, eventEmitter, err) {
   });
 
   if (items.length === 0) {
-    eventEmitter.emit('error', err);
+    try {
+      eventEmitter.emit('error', err);
+    }
+    catch(emit_error){
+      self.logger.warn(`Multiparty failed to emit an 'error' event in 'errorEventQueue' [cause = ${error_message(emit_error)}]`);
```

Comment:
I wonder if this should also somehow communicate the contents of `err` - otherwise that might get lost.

Reply:
I did have a log message for 'err' but found it was always identical to 'emit_error'